tests/quickcheck.rs

```diff
@@ -0,0 +1,302 @@
+#![cfg(feature = "quickcheck")]
```

Comment:
Is quickcheck `#![no_std]` compatible? If not, the reliance on `std` should be included `Cargo.toml` as well as here.

Reply:
Since the `shrink` method returns a `std::boxed::Box`, I'm gonna say no. Fixed, thanks!